developersample/build.gradle

```diff
@@ -0,0 +1,52 @@
+/*
+ * // Copyright (c) Microsoft Corporation.
+ * // All rights reserved.
+ * //
+ * // This code is licensed under the MIT License.
+ * //
+ * // Permission is hereby granted, free of charge, to any person obtaining a copy
+ * // of this software and associated documentation files(the "Software"), to deal
+ * // in the Software without restriction, including without limitation the rights
+ * // to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+ * // copies of the Software, and to permit persons to whom the Software is
+ * // furnished to do so, subject to the following conditions :
+ * //
+ * // The above copyright notice and this permission notice shall be included in
+ * // all copies or substantial portions of the Software.
+ * //
+ * // THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * // IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * // FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * // AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * // THE SOFTWARE.
+ */
+
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
+
+    defaultConfig {
+        applicationId "com.microsoft.identity.client.developersample"
+        minSdkVersion 21
+        targetSdkVersion 25
+        versionCode 1
+        versionName "1.0"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    compile project(':msal')
+    compile 'com.android.support:appcompat-v7:25.2.0'
+    compile 'com.android.volley:volley:1.0.0'
```

Comment:
Friendly reminder: you may need to file a CELA usage notice saying 'I'm taking this oss dependency'